testapp/README.md

```diff
@@ -4,22 +4,31 @@ Some inputs and associated outputs. When testing modifications, you should have
 
 ```txt
 $ MATH_ORGANISATION_NAME=ditrit go run main.go
+PersistentPreRunE
```

Comment:
I don't understand what that could mean.
do you want to see, `PersistentPreRunE` is the log if user used this mecanisme ?
Same for all this file

Reply:
I removed the calls to the pre-run and post-run, it is not really needed. It was meant to show the "new" features.